guide/matrix-vs-discord.md

```diff
@@ -74,32 +76,37 @@ Note that Matrix does not (and cannot, due to its decentralized nature) paywall
+| Bans | Bans are only visible to "server" moderators. One can also see their own ban when they attempt to join a "server", but not the ban reason. | Bans are public to all members, along with the reasons. |
+| Disabled and deleted account handling | Disabling an account is reversible. Messages sent to a "server" from deleted accounts stay unless explicitly removed by somebody else. User name and user settings from deleted accounts are removed. "Servers" owned by accounts shut down by T&S may or may not stay, depending on the exact cause of the shutdown. "Servers" owned by a manually deleted account are automatically shut down. | Disabling an account is usually irreversible. Messages from disabled accounts are not sent to further users and servers. Rooms created by disabled accounts stay. |
 | **Running a bot** | Running a bot in more than 100 "servers" requires proof of identity. Selfbotting is forbidden. | You can run bots on any user accounts[^5] [^6]. Selfbotting is permitted (but be nice). |
-| **Apps to access platform** | You are only allowed to use the official Discord app (including its PTB and Canary variants). Client modifications are forbidden but effectively tolerated. Third-party clients are forbidden. | Element is the main app, but [**you're welcomed to use whatever you wish**](../#what-app-should-i-use). You can even make an app yourself[^6]! |
+| **Apps to access platform** | By ToS, you are only allowed to use the official Discord app (including its PTB and Canary variants). This is not enforced consistently: some modifications and some third party clients are generally tolerated, while others may be forbidden. Third party clients that do not touch certain user profile related features tend to be not caught. | Element is the main app, but [**you're welcomed to use whatever you wish**](../#what-app-should-i-use). You can even make an app yourself[^6]! |
+| **Network access** | **IPv4 only**. Users and bot developers have raised this issue repeatedly, and staff stated this is not urgent at all, despite the fact that IPv4 addresses have been exhausted worldwide: "We of course care of the future of the platform however ipv4<->ipv6 connectivity is not really going away any time soon with the availability of nat64 - and there are much more important things to work on." | **IPv4&v6 support varies depending on homeserver**. |
```

Comment:
```suggestion
| **Network access** | **IPv4 only**. Users and bot developers have raised this issue repeatedly, and staff stated this is not urgent at all, despite the fact that IPv4 addresses have been exhausted worldwide: "We of course care of the future of the platform however ipv4<->ipv6 connectivity is not really going away any time soon with the availability of nat64 - and there are much more important things to work on." | **Most if not all homeservers participating in the public federation have IPv4 connectivity but IPv6 connectivity varies from homeserver to homeserver.**. |
```

Clarified that Homeservers generally speaking must support v4 to federate properly but also are free to support v6 and some do. If you would be v6 only you would limit your self to a large degree due to that matrix is full mesh.